rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -750,62 +744,22 @@ public FileInfo load(String s) throws Exception
     public void onRemoval(RemovalNotification<String, FileMetadata> notification)
     {
       FileMetadata md = notification.getValue();
-      try {
-        md.closeAndCleanup(notification.getCause(), fileMetadataCache);
-        if (!isValidatingCachingBehavior(md.getRemotePath())) {
-          switch (notification.getCause()) {
-            case EXPLICIT:
-              cacheInvalidationCount.inc();
-              break;
-            case SIZE:
-              cacheEvictionCount.inc();
-              break;
-            case EXPIRED:
-              cacheExpiryCount.inc();
-              break;
-            default:
-              break;
-          }
+      md.closeAndCleanup(notification.getCause(), fileMetadataCache);
```

Comment:
Can you add the logs in `FileMetada.closeAndCleanup` when `mdFile.delete()` or `localFile.delete()` return false

Reply:
Done